homework3/src/main/java/HostDevice.java

```diff
@@ -0,0 +1,65 @@
+public class HostDevice {
+
+
+    public ElectricityConsumer[] consumers = new ElectricityConsumer[10];
```

Comment:
why are some objects public and others package private? please document the reason why you need this. otherwise use private.